doc/source/release-notes/Percona-Server-5.5.43-37.2.rst

```diff
@@ -0,0 +1,31 @@
+.. rn:: 5.5.43-37.2
+
+==============================
+ |Percona Server| 5.5.43-37.2
+==============================
+
+Percona is glad to announce the release of |Percona Server| 5.5.43-37.2 on May 8th, 2015. Downloads are available `here <http://www.percona.com/downloads/Percona-Server-5.5/Percona-Server-5.5.43-37.2/>`_ and from the :doc:`Percona Software Repositories </installation>`.
+
+Based on `MySQL 5.5.43 <http://dev.mysql.com/doc/relnotes/mysql/5.5/en/news-5-5-43.html>`_, including all the bug fixes in it, |Percona Server| 5.5.43-37.2 is now the current stable release in the 5.5 series. All of |Percona|'s software is open-source and free, all the details of the release can be found in the `5.5.43-37.2 milestone at Launchpad <https://launchpad.net/percona-server/+milestone/5.5.42-37.2>`_. 
+
+Bugs Fixed
+==========
+
+ |Percona Server| binary tarballs could not be installed (without workaround) due to ``libssl.so.6`` dependency. Different *OpenSSL* versions available in supported distributions caused an error during the installation. This was fixed by replacing the single binary tarball with multiple tarballs depending on the *OpenSSL* library available in the distribution: 1) debian7 - for all *Debian/Ubuntu* versions except *Squeeze/Lucid* (``libssl.so.1.0.0 => /usr/lib/x86_64-linux-gnu/libssl.so.1.0.0 (0x00007f2e389a5000)``); 2) debian6 - only for *Debian Squeeze* and *Ubuntu Lucid* (``libssl.so.0.9.8 => /usr/lib/libssl.so.0.9.8 (0x00007f9b30db6000)``); 3) centos6 - for *CentOS 6* and *CentOS 7* (``libssl.so.10 => /usr/lib64/libssl.so.10 (0x00007facbe8c4000)``); 4) centos5 - to be used only for *CentOS 5* (``libssl.so.6 => /lib64/libssl.so.6 (0x00002aed5b64d000)``). Bug fixed :bug:`1172916`.
+ 
+ Regression introduced by backporting fix for bug :bug:`1319904` would cause installation using ``mysql_install_db`` to fail with ``Error in my_thread_global_end(): 1 threads didn't exit`` error. Bug fixed :bug:`1402074`.
```

Comment:
mysql_install_db would make the server produce an "Error in my_thread_global_end(): 1 threads didn't exit" error message. While this error does not prevent mysql_install_db from completing successfully, its presence might cause any mysql_install_db-calling script to return an error as well. This is a regression introduced ...